tests/tests_fabric/accelerators/test_tpu.py

```diff
@@ -11,30 +11,34 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License
+import os
+
 import pytest
 
 from lightning.fabric.accelerators.tpu import TPUAccelerator
 from tests_fabric.helpers.runif import RunIf
 
 
+@RunIf(tpu=True)
```

Comment:
fine with me, but isn't there a way to test this without running on the TPU?

Reply:
Not really because we would need to mock `TPU.auto_device_count` (it calls torch_xla inside) and then that would defeat the purpose of the test, as we want to test its implementation